hotosm/fAIr
# Bug Fix : Dataset Size

## What does this PR do ? 
- Fixes the bug that introduced after we move away from the raw folder structure to .tar method 
- Fixes #245 

## Consideration : 
I have decided to use chips_length to make it meaningful instead of dataset_size in the database as we have dataset model already and it might create confusion  
- Rebased on top of #250 

## How to test ? 
- Run training and it should have a new field called chips_length after you call the api /training/{training_id}
